docs/tutorial/01-mlrun-basics.ipynb

```diff
@@ -334,15 +334,15 @@
    "source": [
```

Comment:
<p>You can also schedule jobs from the dashboard. On the jobs and monitoring project page, you can create a new job using the&nbsp;<strong>New Job</strong>&nbsp;wizard.&nbsp;At the end of the wizard you can set the job scheduling. In the following example, the job is set to run every 30 minutes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>